helm/nim-llm/values.yaml

```diff
@@ -277,13 +276,14 @@ metrics:
 model:  # most of these values only matter if not using customCommand
   nimCache: /model-store
   name: my-model # optionsl name of the model in the OpenAI API -- used in `helm test`
-  ngcAPISecret: ngc-api
+  ngcAPISecret: "ngc-api"
```

Comment:
We still need to conform to a standard name for this

Reply:
I didn't need to quote it, and it was nearly a mistake that I did :)